### DISCUSSION TITLE
How to use vimium-c to change url from github.com/<anytext> to github.dev/<anytext>

### DISCUSSION THREAD

**Discussion by UriZafrir** [gdh1995/vimium-c#996] [Q&A]
Hi all
I am using vimium C
since I use github.dev to open a vscode in github urls often, I would like to use vimium-c to create a shortcut to change the ".com" to ".dev".
for example I want to change:
to
https://github.dev/gdh1995/vimium-c
I've seen 
https://github.com/gdh1995/vimium-c/discussions/885 
and
https://github.com/gdh1995/vimium-c/wiki/Substitute-URLs-and-text-during-commands

but the documentation is not to easy for me to understand.
For example it's not clear to me which prefix letter should I use:
"The prefix character may be c|p|s|i|g …; "c" means copying, "p" for pasting and "s" for both. "

this is the sed expression I need:
's/github\.com/github.dev/'

I guess maybe something in the lines of this?
r@^github\.com/github\.dev/
but how do I bind it to a key?

Thanks in advance.

**Comment by gdh1995** [comment]
The "prefix character" can be other A-Z letters, or an English word starting with `_` like `_dev`.

Those rules with a prefix character of `s` / `c` / `p` will be auto used in some commands of Vimium C.

To use a rule explicitly, you may try:
1. add a key mapping of `map xxx openUrl url="$S" url_mask sed="_dev"`
2. add a text substitution rule of `_dev/github.com/github.dev/`
